src/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -9,6 +9,12 @@ sidebar_order: 1000
 
 If you need help solving issues with your Sentry JavaScript SDK integration, you can read the edge cases documented here. If you need additional help, you can view our forums, and customers on a paid plan may also contact support.
 
+## Updating to a new Sentry SDK Version
+
+If you update your Sentry SDK to a new major version, you might encounter breaking changes that need some adaption on your end.
+Check out our [migration guide](https://github.com/getsentry/sentry-javascript/blob/7.x/MIGRATION.md) to learn everything you need
```

Comment:
What do you think about pointing that link to the current file (ie the one that's on master)? I think this section won't really matter until we merged v7 anyways. And as far as the beta goes I think it's enough if we put a link in the corresponding github release.

Reply:
I think people might check out troubleshooting if they get stuck with the breaking changes. I think that either we should link in both places to the `master` or to the `7.x` version. I think we can use for now the `7.x` version and then adjust the links once we have a stable v7 release.  If we use the 7.x version once, we have to update docs anyway. But I have no strong feelings about it, so I'm happy to change it either way. 

As you said, having the 7.x link in the GH changelog (which I think we do?) is probably most important for now.

Reply:
I'll let you make the call :)

Reply:
Gonna leave it as is for now and I'll add a remark in our v7 docs release subtask to change the URL